### ISSUE TITLE
option to close tab upon saving

### ISSUE THREAD

**Issue by drook** [gyng/save-in#68]
Please consider the possibility of adding the option to close the tab upon saving the object [matching the routing and saving rules].

Thanks.
P.S. Awesome addon by the way, thanks a lot.

**Comment by gyng** [comment]
Seems useful, I'll look about getting this implemented.

**Comment by gyng** [comment]
A basic version of this is now in v3.4.0.

It closes when the download is activated and not on success, but should suffice for now.

**Comment by drook** [comment]
Thanks a lot !

**Comment by Wombat2018** [comment]
Is it possible to close the tab from which something is being downloaded in general? Or to define at which pages the tab should be closed after using Save in?

**Comment by gyng** [comment]
@Wombat2018 It should be possible, I'll look into it.

**Comment by Wombat2018** [comment]
@gyng 
Is there any news regarding my request?

**Comment by gyng** [comment]
@Wombat2018 Sorry, I lost track of this. I'll open a new issue for this. I've been busy recently so this probably will have to wait a bit more.

**Comment by gyng** [comment]
Implemented in [Save In 4.1.0](https://github.com/gyng/save-in/releases/tag/v4.1.0), including the page-specific follow-up. Add `(after: close-tab)` to a folder line, for example `Read later // (after: close-tab)`, to close the source tab for saves through that destination. Or add `after: close-tab` to a routing rule with a `pageurl:` condition to limit it to particular pages. The tab closes only after the browser accepts the save. Closing as implemented.
